config/samples/watcher_v1beta1_watcher_tls_certificates.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-internal-svc
+spec:
+  dnsNames:
+  - watcher-internal.openstack.svc
```

Comment:
I see the automatically generated certificates for internal-svc and public-svc for other services also add dnsName with .cluster.local, as keystone-internal.watcher-kuttl-default.svc.cluster.local

Reply:
true, adding it